remote-facade/src/main/java/com/iluwatar/remotefacade/domain/Domain.java

```diff
@@ -0,0 +1,23 @@
+package com.iluwatar.remotefacade.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class acting as a database to store details in an Arraylist.
+ */
+public class Domain {
+  public static List<Customer> customers = new ArrayList<Customer>();
+  /**
+  * get the details of the customers in the array to ensure correctness.
+  */
+
+  public static ArrayList<String> getCustomers() {
```

Comment:
Why do we need this function?

Reply:
It was a way for me to check all the details were getting added correctly, I forgot to remove it, but it should be fixed now